src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -220,7 +220,11 @@ public async Task Read()
                 {
                     totalCount = stats.Count;
                 }
-
+                else
+                {
+                    totalCount = rangeResult?.Entries.Length;
```

Comment:
do we really want that? What about paging then? I think it is better to leave it as null?

Reply:
refactored back - leaving null in this case - 
& refactored studio code to handle that